packages/react-storage/src/components/StorageBrowser/actions/handlers/__tests__/listLocations.spec.ts

```diff
@@ -63,6 +64,14 @@ describe('listLocationsHandler', () => {
     );
     expect(result.nextToken).toBeUndefined();
     expect(mockListCallerAccessGrants).toHaveBeenCalledTimes(1);
+    expect(mockListCallerAccessGrants).toHaveBeenCalledWith({
```

Comment:
nit: i dont think test like this where we are call with an input config makes much sense unless we are manipulating some things along the way. like ex, im calling multiple pages, then on recursion `nexTToken` will change then we will want to check if it is using a newNextToken. Something like that IMO. Not gonna hurt but should avoid tests like this

Reply:
Discussed async, Adds value to check if underlying service call is made with the correct params if there's additional logic in between. In this case all action handlers are directly calling the service underneath.

We could potentially remove `toHaveBeenCalledWith` for all action handlers

Reply:
EDIT:
Discussed async, since we're moving things around and in future these handlers could hold additional logic, it makes sense to have this check